ec2/ec2test/subnets_test.go

```diff
@@ -0,0 +1,127 @@
+//
+// goamz - Go packages to interact with the Amazon Web Services.
+//
+//   https://wiki.ubuntu.com/goamz
+//
+// Copyright (c) 2015 Canonical Ltd.
+//
+
+package ec2test_test
+
+import (
+	. "gopkg.in/check.v1"
+
+	"gopkg.in/amz.v3/ec2"
+)
+
+func (s *S) TestAddSubnet(c *C) {
+	subnet, err := s.srv.AddSubnet(ec2.Subnet{})
+	c.Assert(err, ErrorMatches, "empty VPCId field")
+	c.Assert(subnet, DeepEquals, ec2.Subnet{})
+	s.assertVPCsExist(c, true)
```

Comment:
Fair enough, will add a comment and move the assert (which just verifies AddDefaultVPCAndSubnets was called in SetUpTest) before AddSubnet.